analytics_dashboard/templates/footer.html

```diff
@@ -3,11 +3,26 @@
 
 {#  Translators: Application here refers to the web site/application being used (e.g. the dashboard). #}
 <footer class="footer" aria-label="{% trans "Application" %}">
-  {% if show_feedback %}
-  <div class="feedback">
-    <div class="container">
-      <div class="row">
 
+  <div class="container">
+    <div class="row colophon">
+      <nav class="nav-colophon" aria-label="${_('About')}">
+        <ol>
```

Comment:
The LMS actually has these hard-coded in https://github.com/CredoReference/edx-platform/blob/master/lms/djangoapps/branding/api.py#L169

I'll see what I can do about moving the values to somewhere else